<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UbuntuPackagingWidget</class>
 <widget class="QWidget" name="UbuntuPackagingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="Line" name="line_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #DD4818</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>3</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="rightMargin">
      <number>9</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>26</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ubuntu Packaging</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgba(0,0,0,0.2)</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>20</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget_2">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="pageAvailable">
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="topMargin">
          <number>6</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonReload">
           <property name="text">
            <string>Reload</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonReset">
           <property name="text">
            <string>Reset to defaults</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page_package">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <spacer name="horizontalSpacer_6">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>500</width>
                 <height>0</height>
                </size>
               </property>
               <property name="title">
                <string>Click</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <widget class="QTabWidget" name="tabWidget">
                  <property name="currentIndex">
                   <number>0</number>
                  </property>
                  <widget class="QWidget" name="tabSimple">
                   <attribute name="title">
                    <string>Simple</string>
                   </attribute>
                   <layout class="QGridLayout" name="gridLayout_2">
                    <item row="8" column="0">
                     <widget class="QLineEdit" name="lineEdit_description"/>
                    </item>
                    <item row="1" column="0">
                     <widget class="QLabel" name="label">
                      <property name="text">
                       <string>Name</string>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="0" colspan="2">
                     <widget class="QLineEdit" name="lineEdit_name">
                      <property name="text">
                       <string/>
                      </property>
                     </widget>
                    </item>
                    <item row="5" column="0">
                     <widget class="QLabel" name="label_4">
                      <property name="text">
                       <string>Title</string>
                      </property>
                     </widget>
                    </item>
                    <item row="3" column="0">
                     <widget class="QLabel" name="label_3">
                      <property name="text">
                       <string>Maintainer</string>
                      </property>
                     </widget>
                    </item>
                    <item row="4" column="0" colspan="2">
                     <widget class="QLineEdit" name="lineEdit_maintainer">
                      <property name="text">
                       <string/>
                      </property>
                     </widget>
                    </item>
                    <item row="6" column="0" colspan="2">
                     <widget class="QLineEdit" name="lineEdit_title">
                      <property name="text">
                       <string/>
                      </property>
                     </widget>
                    </item>
                    <item row="10" column="0">
                     <widget class="QLineEdit" name="lineEdit_version"/>
                    </item>
                    <item row="9" column="0">
                     <widget class="QLabel" name="label_6">
                      <property name="text">
                       <string>Version</string>
                      </property>
                     </widget>
                    </item>
                    <item row="11" column="0" colspan="2">
                     <widget class="QGroupBox" name="groupBox_2">
                      <property name="title">
                       <string>Security Policy Groups</string>
                      </property>
                      <layout class="QVBoxLayout" name="verticalLayout">
                       <item>
                        <widget class="QListWidget" name="listWidget">
                         <property name="alternatingRowColors">
                          <bool>true</bool>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <layout class="QHBoxLayout" name="horizontalLayout">
                         <item>
                          <widget class="QPushButton" name="pushButton_addpolicy">
                           <property name="text">
                            <string>+</string>
                           </property>
                          </widget>
                         </item>
                        </layout>
                       </item>
                      </layout>
                     </widget>
                    </item>
                    <item row="7" column="0">
                     <widget class="QLabel" name="label_7">
                      <property name="text">
                       <string>Description</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                  <widget class="QWidget" name="tabManifest">
                   <attribute name="title">
                    <string>Manifest</string>
                   </attribute>
                   <layout class="QVBoxLayout" name="verticalLayout_6">
                    <item>
                     <widget class="QPlainTextEdit" name="plainTextEditJson"/>
                    </item>
                   </layout>
                  </widget>
                  <widget class="QWidget" name="tabAppArmor">
                   <attribute name="title">
                    <string>AppArmor</string>
                   </attribute>
                   <layout class="QVBoxLayout" name="verticalLayout_6">
                    <item>
                     <widget class="QPlainTextEdit" name="plainTextEditAppArmorJson"/>
                    </item>
                   </layout>
                  </widget>
                  <widget class="QWidget" name="tabExcludes">
                   <attribute name="title">
                    <string>Excludes</string>
                   </attribute>
                   <layout class="QVBoxLayout" name="verticalLayout_8">
                    <item>
                     <widget class="QPlainTextEdit" name="plainTextEdit_excludes"/>
                    </item>
                   </layout>
                  </widget>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButtonClickPackage">
                  <property name="text">
                   <string>Create package</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_init">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QProgressBar" name="progressBar">
             <property name="maximum">
              <number>0</number>
             </property>
             <property name="value">
              <number>-1</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageNotAvailable">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>227</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Click packaging is only available for QML projects.</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_8">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>227</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
